modules/ROOT/pages/notifications.adoc

```diff
@@ -0,0 +1,19 @@
+= Important Notifications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Use these sources to stay informed about new releases and downtime for Anypoint API Community Manager.
+
+== Release Announcements
+
+Keeping the API Community Manager components updated is critical to keep your API community operating smoothly and optimally. Watch for new releases in these channels so you can plan to xref:update-acm.adoc[update your installation] at your convenience.
```

Comment:
To what does "in these channels" refer? Do you mean "the following channels" (which indicates a list)?
If this refers to the Scheduled Maintenance section and the RNs, why not just name them?
```suggestion
Updating API Community Manager components is critical to keeping your API community operating smoothly and optimally. Watch for new releases in these channels so that xref:update-acm.adoc[update your installation].
```

Reply:
yes this refeers to the scheduled maintenance and release notes - agree with naming them directly.

agree with your other suggestions.

Reply:
Good point. Edited to name the scheduled maintenance and release note pages and integrate the wording improvements.